tests/Framework/ConstraintTest.php

```diff
@@ -644,10 +644,8 @@ public function isEqualProvider()
 -b\\n
 +p\\n
 @@ @@
- i\\n
 -j\\n
 +w\\n
- k'
```

Comment:
new default of `diff` no longer outputs lines that are the same at the end of the diff, and if there are more than X consecutive unchanged lines (5 IIRC)